src/main/java/org/broadinstitute/hellbender/utils/test/IntegrationTestSpec.java

```diff
@@ -94,17 +92,20 @@ public void executeTest(final String name, CommandLineProgramTest test) throws I
             tmpFiles.add(fl);
         }
 
-        final String args = String.format(getArgs(), tmpFiles.toArray());
+        final String preFormattedArgs = getArgs();
+        final String formattedArgs = String.format(preFormattedArgs, tmpFiles.toArray());
```

Comment:
Can you explain why `String.format()` doesn't throw here? According to the docs it should throw an `IllegalFormatException`